src/dbus/common/types.hpp

```diff
@@ -719,6 +719,33 @@ struct TrelInfo
     TrelPacketCounters mTrelCounters; ///< The TREL counters.
 };
 
+struct BorderAgentCounters
+{
+    uint32_t mEpskcActivations;              ///< The number of activations
+    uint32_t mEpskcDeactivationClears;       ///< The number of deactivations due to cleared via API
+    uint32_t mEpskcDeactivationTimeouts;     ///< The number of deactivations due to timeout
+    uint32_t mEpskcDeactivationMaxAttempts;  ///< The number of deactivations due to max connection attempts
+    uint32_t mEpskcDeactivationDisconnects;  ///< The number of deactivations due to commissioner disconnected
+    uint32_t mEpskcInvalidBaStateErrors;     ///< The number of activation failures caused by invalid border agent state
+    uint32_t mEpskcInvalidArgsErrors;        ///< The number of activation failures caused by invalid argument
+    uint32_t mEpskcStartSecureSessionErrors; ///< The number of activation failures due to start secure session error
+    uint32_t mEpskcSecureSessionSuccesses;   ///< The number of successful secure session establishment
+    uint32_t mEpskcSecureSessionFailures;    ///< The number of failed secure session establishement
+    uint32_t mEpskcCommissionerPetitions;    ///< The number of active commissioner petitioned
+    uint32_t mPskcSecureSessionSuccesses;    ///< The number of successful secure session establishment
+    uint32_t mPskcSecureSessionFailures;     ///< The number of failed secure session establishement
+    uint32_t mPskcCommissionerPetitions;     ///< The number of active commissioner petitioned
+    uint32_t mMgmtActiveGets;                ///< The number of MGMT_ACTIVE_GET.req received
```

Comment:
How about it is named as xxxGetCounts in here and the proto to have the alignment? Get is not a noun, while Count is a noun, better to use Noun in the ending? Similar to the below one field PENING_GET.

Reply:
Removed this struct definition as it's only needed when adding a dbus getBorderAgentInfo api.

For the proto file as the fields are already under `BorderAgentCounters`, it feels redundant to add counts in the field names. `MgmtActiveGet/MgmtPendingGet` is the message name, I added `_req` to the end of `mgmt_active_gets` and `mgmt_pending_gets` so the names also end with nouns.